azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -56,10 +56,10 @@ public Emailer(final Props props) {
     this.mailSender = props.getString("mail.sender", "");
     this.tls = props.getString("mail.tls", "false");
 
-    final int mailTimeout = props.getInt("mail.timeout.millis", 10000);
+    final int mailTimeout = props.getInt("mail.timeout.millis", 30000);
     EmailMessage.setTimeout(mailTimeout);
     final int connectionTimeout =
-        props.getInt("mail.connection.timeout.millis", 10000);
+        props.getInt("mail.connection.timeout.millis", 30000);
```

Comment:
Yes, it will try 3 times with 30 sec timeout interval, so the total would be 90 sec.